courses/javascript/task/yasir.js

```diff
@@ -0,0 +1,10 @@
+var firstname = "Yasir";
+var lastname = " Mohammed";
+
+var fullname =firstname + lastname;
+
+var location= " Vijayawada";
```

Comment:
```suggestion
var location= "Vijayawada";
```

Remove starting space